.gitignore

```diff
@@ -50,6 +50,7 @@ null
 slices.json
 **/.antlr
 /joern-cli/frontends/csharpsrc2cpg/bin
+/joern-cli/frontends/rubysrc2cpg/type_stubs
```

Comment:
I wonder for this rule and the one above, to instead just do
```
/joern-cli/frontends/*/bin
/joern-cli/frontends/*/type_stubs
```

Reply:
I'll make a note to do this in the next PR, want to merge this one in so long so that I can start on fixing the DF tests.